src/Simulation/EntryPointDriver/Simulation.cs

```diff
@@ -53,6 +53,8 @@ public static class Simulation<TCallable, TIn, TOut> where TCallable : AbstractC
                         ? (false, () => new QuantumSimulator())
                         : simulator == settings.ToffoliSimulatorName
                         ? (false, new Func<IOperationFactory>(() => new ToffoliSimulator()))
+                        // : simulator == settings.SparseSimulatorName
```

Comment:
I'm a bit confused why Sparse Simulator disappeared. Is it true that QuantumSimulator can be one of the two? Probably some comment would help.

Reply:
I commented out every line about SparseSimulatorName in the EntryPointDriver until I resolve the [following TODO](https://github.com/microsoft/qsharp-runtime/pull/918/files#r792327069):
```c#
// TODO(rokuzmin): Requires support in Q# compiler? 
// Is invoked from the .cs code generated from .qs, and the arg for this param is not provided.
```